src/tslint.json

```
@@ -51,7 +51,7 @@
     "no-eval": true,
     "no-misused-new": true,
     "no-non-null-assertion": true,
-    "no-shadowed-variable": true,
+    "no-shadowed-variable": false,
```

Dejar en true y resolver la(s) variable(s) "shadowed" que hayan aparecido ...

```suggestion
    "no-shadowed-variable": true,
```

Se dice que una variable quedó "shadowed", "oculta" cuando queda cubierta por otra, ejemplo/típicamente, cuando una variable local tiene el mismo nombre que una variable global. Esto dificulta la detección y corrección de errores, razón por la que el linter lo advierte y bloquea

Entre [otras explicaciones y formas de resolverlo](https://stackoverflow.com/a/54105345/3180052), la típica es simplemente cambiarle el nombre a la variable que el linter está indicando como _shadowed_, bien sea a la de ámbito local (recomendado) o, a la de ámbito global